.github/copilot/integrate-social-share-button.prompt.md

```
@@ -1,653 +1,261 @@
-  steps for CRA, Next.js App Router, Next.js Pages Router, Vite/Vue/Angular,
-  Preact, Qwik, and Vanilla HTML. Use this skill whenever a developer asks how to add the
-  share button to their project.
+  Automatically integrate @aossie-org/social-share-button into a client web project
+  with zero manual steps except specifying where to place the button.
 ---
 
 # SocialShareButton — Integration Skill
 
-You are helping a developer integrate the **SocialShareButton** library
-(`social-share-button-aossie`) into their project. Follow these rules strictly.
+## Goal
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)